docs/team/dionegoh.md

```diff
@@ -11,42 +11,55 @@ It is written in Java and has over 10kLoC.
-    * What it does: Allows users to delete a friend from the address book by typing in the relevant command into the GUI. 
-    * Justification:
+2. **New Feature: command aliases**
+   * What it does: Enables users to enter shorter command aliases instead of the usual command.
+   * Justification: Allows experienced users familiar with the commands to enter commands faster.
+   
+3. **Enhancement to existing feature `addfriend`**: 
+    * What it does: Allows users to add a friend into address book by typing in the relevant command into the GUI.
     * Highlights:
-    * Credits: 
+      * Added a new `Description` field for a `Friend`
+      * Made other fields in optional and only `FriendName` compulsory, so that users who do not know full details of a friend
+        can still add the friend into `Amigos`.
+      
+4. **Enhancement to existing feature `deletefriend`**: 
+    * What it does: Allows users to delete a friend from `Amigos`.
+    * Highlights: Users can delete a friend either by entering the `FriendName` or a valid `Index`. 
+      Users can delete an existing friend by `FriendName` even if the
+      friend is not currently being shown on the filtered list on the GUI. 
 
-* **editfriend**: Added the functionality to allow user to edit details of a friend from the address book.
-    * What it does: Allows users to edit a friend from the address book by typing in the relevant command into the GUI.
-    * Justification:
-    * Highlights:
-    * Credits:
+5. **Enhancement to existing feature `editfriend`**:
+    * What it does: Allows users to edit a friend from `Amigos`.
+    * Justification: Users can edit a friend either by entering the `FriendName` or a valid `Index`.
+    * Highlights: The prefix `cn/` is used to identify the current `FriendName`. `nn/` is used for the
+      edited `FriendName`.
+
+6. **Enhancement to existing feature `findfriend`**:
+    * What it does: Allows users to search for friends from `Amigos` using keywords.
+    * Highlights: Keywords are not limited to `FriendName`. `Tag` and `LogName` of a Person are also
+      searched. The user input is split according to the prefixes (`n/` for `FriendName`, `t/` for `Tag`,
+      `ttl` for `LogName`). Also, keywords do not have to be matched fully. As long as a substring is matched, then 
+      the friend would be shown.
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=dionegoh&breakdown=true)
 
 * **Project management**:
     * Update site-wide settings [\#21](https://github.com/AY2122S2-CS2103-F09-2/tp/pull/21)
-
-* **Enhancements to existing features**:
-    *
-
+    
 * **Documentation**:
     * User Guide:
-        * Added documentation for the features `addfriend`, `deletefriend`, and `editfriend`' 
-      [\#22](https://github.com/AY2122S2-CS2103-F09-2/tp/pull/22)
+        * Added documentation for the features `addfriend`, `deletefriend`, and `editfriend`', `showfriend`,
```

Comment:
listfriend is mentioned twice

also, should be listfriends instead of listfriend